serialization.md

```diff
@@ -167,7 +167,8 @@ Here is an example image JSON file:
             <li>amd64</li>
             <li>arm</li>
         </ul>
-        More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation.
+        More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation. Future values will be consistent with those already canonicalized by the <a href="https://www.debian.org/ports/">Debian ports list</a>.
```

Comment:
Well Debian doesn't use `386` but `i386`. They do seem to have an architecture named `arm`, there are lost of Debian arm architectures based on the build flags. I don't demand that we follow Debian's lead, but it was the closest thing that I could find to a standard for specifying actual BUILD TIME architectures! I think that it is good to copy someone, even if there are some differences in the naming. I think that it is better than to just come up with random names for things. And actually, I think that in the case of arm, following Debian's standard brings along a lot of valulable technical know-how (AKA, the taxonomy of arm build setups ;) ).